assets/queries/openAPI/parameter_objects_headers_dup_name/query.rego

```diff
@@ -0,0 +1,49 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+	params := doc.paths[n].parameters
```

Comment:
Shouldn't [operations object](https://swagger.io/specification/#operation-object) be checked?